onnxscript/function_libs/torch_aten/ops/nn.py

```diff
@@ -1521,16 +1521,44 @@ def aten_upsample_bicubic2d_backward(
     raise NotImplementedError()
 
 
+@torch_op("aten::upsample_bilinear2d", trace_only=True)
 def aten_upsample_bilinear2d(
     self: TensorType,
-    output_size: INT64,
-    align_corners: bool,
+    size: Optional[INT64] = None,
+    # align_corners: bool,
```

Comment:
Why do we need to comment it out instead of just keeping it there?

Reply:
fixed